### DISCUSSION TITLE
I can't donwload a video

### DISCUSSION THREAD

**Discussion by adrichuela** [aclap-dev/video-downloadhelper#634] [Help and Q&A]
### Your question

This is the message that appear: Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error!

### Addon & CoApp version and details

This is the message that appear: Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error!

**Comment by GrampaWildWilly** [comment]
This is not a useful problem report.  Please read this: #602.  Once you have digested the information over there on how to report a problem, you can update your post above.  Look above your post & to the right.  You'll see 3 dots.  Click them.  I think you'll be able to take it from there.
